magento/magento2
# Fix #28011 - remove page number from query on layered navigation swatch filter

### Description (*)

This change makes swatch layered navigation filter consistent with standard filters, by removing `p` query parameter from filter url in same way as in https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/Model/Layer/Filter/Item.php
Fixes #28011 

### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#28011: ElasticSearch When applying the extra filter to the product listing, Magento keeps the same page number

### Manual testing scenarios (*)

1. Have at least one swatch filter on layered navigation and multiple pages of products
2. Go to last page on PLP
3. Use swatch attribute as filter
4. `p` parameter should be removed from url and user should see first page, same as when non-swatch attribute is used as filter

### Questions or comments


### Contribution checklist (*)
